apps/opik-backend/src/main/java/com/comet/opik/domain/filter/FilterQueryBuilder.java

```diff
@@ -60,7 +60,7 @@ public class FilterQueryBuilder {
                     FieldType.FEEDBACK_SCORES_NUMBER,
                     "has(groupArray(tuple(lower(name), %1$s)), tuple(lower(:filterKey%2$d), toDecimal64(:filter%2$d, 9))) = 1",
                     FieldType.DICTIONARY,
-                    "lower(JSON_VALUE(%1$s, :filterKey%2$d)) = lower(:filter%2$d)")),
+                    "lower(JSON_QUERY(%1$s, :filterKey%2$d)) = lower(:filter%2$d) OR lower(JSON_VALUE(%1$s, :filterKey%2$d)) = lower(:filter%2$d)")),
```

Comment:
I agree it's not the best user experience. However, the operation `[*]` doesn't work correctly in `JSON_VALUE`; instead of getting the whole array, it gets only the first element. I think it would be better to have a way to match an array string instead of not having it at all. But I agree, it's not the best experience